.github/workflows/test.yml

```
@@ -20,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.10'] # Only supported one at the moment
+        python-version: ['3.10', '3.11'] # Only supported one at the moment
```

nit: should this comment be dropped now (like above)?

```suggestion
        python-version: ['3.10', '3.11']
```

---

Yes, I changed it locally but forgot to push.